src/ui/OpenFile/UiReportMenu.js

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright 2022 EPAM Systems, Inc. (https://www.epam.com/)
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @fileOverview UiReportMenu
+ * @author Epam
+ * @version 1.0.0
+ */
+
+import React, { useState } from 'react';
+
+import Screenshot from '../../engine/utils/Screenshot';
+import ViewMode from '../../store/ViewMode';
+import { Tooltip } from '../Tooltip/Tooltip';
+import { UIButton } from '../Button/Button';
+import UiModalInfo from '../Modals/ModalInfo';
+import { useSelector } from 'react-redux';
+
+export const UiReportMenu = () => {
+  const [showModalDicomTags, setShowModalDicomTags] = useState(false);
+
+  const store = useSelector((state) => state);
```

Comment:
For a better understanding and readability, it's better to get the necessary variables from store.
`const {viewMode, graphics2d, volumeRenderer, isLoaded} = useSelector(state => state)`